packages/shared/src/icons/TextIcon.js

```diff
@@ -30,19 +30,20 @@ const styleSheet = StyleSheet.create({
  *
  * We currently support only one font family: "spfont".
  */
-function TextIcon(props: Props) {
-  return (
-    <Text style={[styleSheet.icon, props.style]}>
-      <Translation passThrough={props.code} />
-    </Text>
-  );
+class TextIcon extends React.Component<Props> {
+  static defaultProps = {
+    style: {
+      color: Color.textLight,
+      fontSize: 20,
+    },
+  };
+  render() {
+    return (
+      <Text style={[styleSheet.icon, this.props.style]}>
+        <Translation passThrough={this.props.code} />
+      </Text>
+    );
+  }
```

Comment:
Why is this change needed?

Reply:
This is to be able to use `Animated.createAnimatedComponent`; it was complaining that it only works with classes.